lib/hyper.fnl

```diff
@@ -0,0 +1,72 @@
+(require-macros :lib.macros)
```

Comment:
Personally, I'm not a big proponent of "hyper" key, because 

a) afaik, it cannot be done by just using Hammerspoon, it requires also to install Karabiner
b) instead of relying to complex keybindings it could be done via modals/transients/hydras, essentially, I would prefer to control everything with h/j/k/l

That being said: unless there's a convincing argument for it, I would rather not introduce it at this point.